Documentation/ApiOverview/Configuration/ConfigurationOverview.rst

```diff
@@ -43,10 +43,10 @@ Extension files
 ---------------
 
 :ref:`composer.json <composer-json>`
-   Composer configuration
+   Composer configuration, required in composer-based installations
 
 :ref:`ext_emconf.php <extension-declaration>`
-   (required) Extension declaration
+   Extension declaration, required in legacy installations
```

Comment:
```suggestion
   Extension declaration, required in Legacy installations
```

Reply:
Legacy is not a proper noun, it is just an adjective